src/main/java/org/ow2/proactive/catalog/graphql/handler/catalogobject/CatalogObjectBucketIdFilterHandler.java

```diff
@@ -25,17 +25,14 @@
  */
 package org.ow2.proactive.catalog.graphql.handler.catalogobject;
```

Comment:
If we align the naming, this class should be renamed as CatalogObjectBucketNameFilterHandler.java